client/components/plans/plan/style.scss

```diff
@@ -137,6 +137,10 @@
 	.plan-list .jetpack_premium, .plan-list .jetpack_business {
 		width: 49%;
 	}
+
+	.plan-list .jetpack_premium .gridicons-checkmark-circle, .plan-list .jetpack_business .gridicons-checkmark-circle {
```

Comment:
Its a shame we have to have both product slugs in here, but there's already some refactoring of this happening in https://github.com/Automattic/wp-calypso/pull/2985, so lets :ship: and I'll address it there.